### ISSUE TITLE
the erl run command doesn't include directories marked as dependencies

### ISSUE THREAD

**Issue by vrescobar** [ignatov/intellij-erlang#604]
Hello,

I created a couple of examples based on rebar and I can't get them running when I need runtime dependencies on external modules (jsx in my example).

At build time all my dependencies are placed on the subfolder "deps" and managed directly by rebar (downloaded, compiled and so on), and to get running those basic programs in the command line it is not a problem.

Since they are marked as "root includes" the editor also finds everything it needs for code completion and I can even follow the function calls correctly doing control click.

The only problem is when I want to run the code that the generated "erl" command forgets to include the dependencies:

```bash
/usr/local/Cellar/erlang/17.3/lib/erlang/bin/erl -pa /Users/vresko/projects/erlang_demo/ebin -pa /Users/vresko/projects/erlang_demo -eval e8demo:init(). -s init stop -noshell
{"init terminating in do_boot",{undef,[{jsx,decode,[<<269 bytes>>],[]},{e8demo,init,0,[{file,"src/e8demo.erl"},{line,21}]},{erl_eval,do_apply,6,[{file,"erl_eval.erl"},{line,657}]},{init,start_it,1,[]},{init,start_em,1,[]}]}}

Crash dump was written to: erl_crash.dump
init terminating in do_boot ()
```

I tried to create jsx as a separate project and declare it as a dependant module to include in the main project where I develop (instead of have it nested) and anyway the erl command is also not working at all:

```bash
/usr/local/Cellar/erlang/17.3/lib/erlang/bin/erl -pa /Users/vresko/projects/erlang_libs/jsx -pa /Users/vresko/projects/erlang_demo/ebin -pa /Users/vresko/projects/erlang_demo -eval e8demo:init(). -s init stop -noshell
{"init terminating in do_boot",{undef,[{jsx,decode,[<<269 bytes>>],[]},{e8demo,init,0,[{file,"src/e8demo.erl"},{line,21}]},{erl_eval,do_apply,6,[{file,"erl_eval.erl"},{line,657}]},{init,start_it,1,[]},{init,start_em,1,[]}]}}

Crash dump was written to: erl_crash.dump
init terminating in do_boot ()
```

The only workaround I find at the moment to run my code from the IDE is to add hardcoded in the run dialog "-pa /path/to/project/jsx/ebin/" to force it find the path (os variables don't look to work to indicate $HOME).

I inspected all the options in the "file" -> "Project Structure" and I think I tried any possible option to declare my dependencies, I am not sure what else could I do. I am running on osx with the latest available version of intellij and erlang plugin. Please let me know if there is any debug info I could attach.

**Comment by deadok22** [comment]
Hi @vrescobar,

The run command isn't supposed to add include directories to code path arguments. It only adds output directories of IntelliJ modules which are among dependencies of the module you're running your run configuration from.

So I suppose you do not have your dependencies from under ```deps/``` set up as IntelliJ modules. The plugin will set them up automatically if you import your rebar project via ```Create Project From Existing Sources``` action. 

Try doing this and explore ```Project Structure``` to see how dependencies for your IntelliJ modules are set.

**Comment by vrescobar** [comment]
Hi @deadok22 ,
I made a new project, added the rebar dependencies per hand (rebar.config and in mayapp.app.src) and after fetch sources I could "imported new project from existing sources" as you said and worked properly.

In the old project, the one I reported, had the dependencies set as well, however I assume somewhere I made an error so I removed and re-added matching the structure of the new one and now it looks to work without any problem.

Thank you for your comment!
